rpp-charter.md

```diff
@@ -1,95 +1,55 @@
-[RFC 6570](https://datatracker.ietf.org/doc/html/rfc6570), [RFC 8288](https://datatracker.ietf.org/doc/html/rfc8288),
-[RFC 9421](https://datatracker.ietf.org/doc/html/rfc9421), [RFC 9652](https://www.rfc-editor.org/rfc/rfc9652.html), 
-and the outputs of the [httpapi working group](https://datatracker.ietf.org/wg/httpapi/about/).
+All consensus decisions are conducted on the mailing list.
 
 # Deliverables
 
-This working group will publish on its wiki its operational practices,
-including but not limited to, the adoption of work items, usage of
-tools beyond the IETF’s datatracker, and implementation requirements
-for the promotion of work to the IESG.
-
-This working group will find consensus on a set of core requirements for RPP.
-In alignment with the [IESG's Statement on Support Documents dated 2023-08-24](https://datatracker.ietf.org/doc/statement-iesg-support-documents-in-ietf-working-groups-20230824/),
-these requirements will be publicly available on the working group's wiki.
-Analysis of the functionality included and commonly used in core EPP for domain names, 
-hosts and contacts ([RFC5730](https://datatracker.ietf.org/doc/html/rfc5730),
-[5731](https://datatracker.ietf.org/doc/html/rfc5731), [5732](https://datatracker.ietf.org/doc/html/rfc5732), and
-[5733](https://datatracker.ietf.org/doc/html/rfc5733)) is expected to be created as input material for the requirements. 
-The working group may also consider RPP extensions that are functional equivalents of 
-registered [EPP extensions](https://www.iana.org/assignments/epp-extensions/epp-extensions.xhtml) in
-the construction of requirements.
-
-Once established, these requirements may be changed through explicit working group 
-consensus and, when the chairs determine it to be a major change, in consultation 
-with the group’s responsible Area Director. The finding of consensus for both the 
-initial requirements and any changes will be explicitly noted by the chairs on the 
-working group's mailing list.
-
-After finding consensus regarding requirements, the working group will deliver to the
-IESG documents addressing the following topics in an order determined through agreement
-of the chairs with the responsible Area Director:
-* core architecture, including extension mechanisms
-* specifications necessary to implement the core architecture
-* specifications necessary to implement extensions
-* specifications regarding the provisioning of domain names using RPP
-* mappings between RPP and EPP
-
-The working group may consider work for the provisioning of other types of Internet registries (e.g. IP addresses
-and autonomous system numbers) with consultation from the responsible Area Director.
-
-A written record of acceptance with an explanation of working group scope of I-Ds as working group work items 
-is to be recorded with explicit acknowledgement of the chairs and published on the wiki.
+The RPP WG will work on the following milestones within the scope of the charter:
```

Comment:
if the whole document is now in present tense (which kind of makes sense to me if you look from the perspective of reading it in the future), then this sentence would need consequently also be in present tense.

Reply:
agreed... although I wonder now if my "present tense" idea is a rare position within IETF charters, for example the example here: https://datatracker.ietf.org/doc/html/rfc2418#section-2.2 is future tense.

Reply:
If the charter's purpose is to review what the intention was sometime in the past, then a future tense now would make sense. Having present tense now does read a bit strange but as Andy said earlier, each of these choices can be reasoned.

Reply:
There is apparently some variability on this point in IETF charters, I'll keep the current text for this version, but we can open an issue should a comment on this show up in the future.